scripts/deploy.sh

```diff
@@ -59,7 +59,9 @@ if [[ "${DEPLOY_AIO}" == "yes" ]]; then
   if [[ ! -d /etc/openstack_deploy/ ]]; then
     ./scripts/bootstrap-aio.sh
     # move OSA variables file to AIO location.
-    mv /etc/openstack_deploy/user_variables.yml /etc/openstack_deploy/user_osa_aio_variables.yml
```

Comment:
I think it would be better to overwrite the file otherwise someone testing a change with an AIO might not get what they expected.